apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -74,6 +75,38 @@ const getSkillLevelMessage = (
   return skillLevel;
 };
 
+const getTitle = (poolSkill: PoolSkill | undefined, intl: IntlShape) => {
+  let title = "";
+  const skillLevel = getSkillLevelMessage(poolSkill, intl);
+  if (!isEmpty(skillLevel)) {
```

Comment:
`getSkillLevelMessage()` will never fail the check as it returns a truthy string guaranteed, it either returns a skill level or the fallback `notFound`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/aedbb477-3cf6-4258-9cb1-8f3c691457dc)

Reply:
Oops ..looks like I have put back the old code during merge conflict.